src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/keybinding-vim-uncompressed.js

```diff
@@ -701,6 +701,7 @@ dom.importCssString(".normal-mode .ace_cursor{\
       dialog.className = "ace_dialog ace_dialog-bottom";
     else
       dialog.className = "ace_dialog ace_dialog-top";
+    dialog.setAttribute("spellcheck", "false");
```

Comment:
This seems like a duplicate of the line below, does it have a purpose?

Reply:
Not sure how that snuck in there :-/